m/pager.go

```diff
@@ -44,6 +44,10 @@ type Pager struct {
 
 	// NewPager shows lines by default, this field can hide them
 	ShowLineNumbers bool
+
+	// If true, pager will clear the screen on return. If false, pager will
+	// clear the last line, and show the cursor.
+	DeInit bool
```

Comment:
`DeInit` reflects what it does internally, maybe something like `RestoreScreenWhenDone` would reflect better what effect a user would see from changing it?

Not sure what a great name would be, if you don't want to change it I'm fine with `DeInit` for now.